source/channels/favorite-channels.rst

```diff
@@ -18,4 +18,6 @@ This adds the channel to a **Favorites** list at the top of the channel sidebar,
 .. image:: ../images/favorites-list-sidebar.png
     :alt: Favorite channels in the channel sidebar.
        
-**On mobile:** Select the channel name at the top of the center pane to access the drop-down menu, then select **Favorite**. To remove a channel from the **Favorites** list, select **Favorite** again.
\ No newline at end of file
+**On mobile:** Select the channel name at the top of the center pane to access the drop-down menu, then select **Favorite**. To remove a channel from the **Favorites** list, select **Favorite** again.
+.. image:: ../images/fav.gif
+  :alt: Favorite channels in the channel sidebar using phone app.
```

Comment:
Is this imagine from the mobile app? It looks like it's from the desktop app.

Reply:
yup that is from mobile app... what chenges do i need to make ?

Reply:
I'd propose replacing "phone app" with "mobile app". :)